### PR TITLE
Test align and distribute editor actions

### PR DESCRIPTION
**Pull request by nikku**
Depends on https://github.com/bpmn-io/diagram-js/pull/537.

### CODE REVIEW

File: test/spec/features/editor-actions/BpmnEditorActionsSpec.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (philippfromme) [inline test/spec/features/editor-actions/BpmnEditorActionsSpec.js:1]: Accidental empty line? 🤡

File: test/spec/features/editor-actions/BpmnEditorActionsSpec.js [hunk 2]
```diff
@@ -91,4 +94,110 @@ describe('features/editor-actions', function() {
...
+        selection.select(elements);
+        editorActions.trigger('alignElements', { type: 'middle' });
+
+        // then
+        expect(elements.map(e => e.y + e.height / 2)).to.eql([ 311, 311, 311 ]);
+      }
+    ));
+
+
+    it('should not align if to few elements', inject(
```
Comment (philippfromme) [inline test/spec/features/editor-actions/BpmnEditorActionsSpec.js:128]:
```suggestion
    it('should not align if too few elements', inject(
```

File: test/spec/features/editor-actions/BpmnEditorActionsSpec.js [hunk 3]
```diff
@@ -91,4 +94,110 @@ describe('features/editor-actions', function() {
...
+        selection.select(elements);
+        editorActions.trigger('distributeElements', { type: 'horizontal' });
+
+        // then
+        expect(elements.map(e => e.x + e.width / 2)).to.eql([ 433, 574, 714 ]);
+      }
+    ));
+
+
+    it('should not distribute if to few elements', inject(
```
Comment (philippfromme) [inline test/spec/features/editor-actions/BpmnEditorActionsSpec.js:181]:
```suggestion
    it('should not distribute if too few elements', inject(
```

**Review by philippfromme — COMMENTED** [review]
Looking good. Only found some tiny issues.

**Comment by nikku** [comment]
All fixed I believe!

**Comment by philippfromme** [comment]
Alright, let's wait for bpmn-io/diagram-js#537 and then merge.

**Comment by nikku** [comment]
Should be good for another review, at last.

**Review by philippfromme — APPROVED** [review]
Looks good! 🦾

**Pull request merged**